package.json

```diff
@@ -36,7 +36,7 @@
     "compile": "npm run compile-es && npm run compile-commonjs",
     "compile-es": "tsc --outDir es --module es6 --moduleResolution node",
     "compile-commonjs": "tsc --outDir ./",
-    "prepublish": "npm run compile",
+    "prepublishOnly": "npm run compile",
```

Comment:
While compiling is not the most thorough set of checks we could run before publishing, this change just gets us up to date with modern npm.

https://docs.npmjs.com/misc/scripts#prepublish-and-prepare